tests/std/tests/Dev10_851347_weak_ptr_virtual_inheritance/test.cpp

```diff
@@ -4,48 +4,109 @@
 #include <assert.h>
 #include <memory>
 #include <utility>
-
 using namespace std;
 
+// Also test GH-1102 "<memory>: weak_ptr conversions don't preserve control blocks for expired objects"
+template <typename T, typename U>

```

Comment:
Should this use `class` for template type parameters to be consistent with the product code, or do you think we should continue to use arbitrary styles in test code?

Reply:
I think we should file an issue about making the test code consistent (I am most notably thinking of const and using-directives). We should also consider typename-ing the product code although that would change a lot of lines.

Reply:
Filed #1110.